### PR TITLE
i18n: Unmark strings

### PR DESCRIPTION
**Pull request by unho**
These are not exposed anywhere.

### CODE REVIEW

File: pootle/templates/includes/formtable.html [hunk 1]
```diff
@@ -12,24 +12,28 @@
...
-		{% blocktrans %}Showing{% endblocktrans %} {{ formtable.results_per_page }} {{ formtable.form.results_per_page.label }}
+		  {% blocktrans with count=formtable.results_per_page trimmed %}
+		  Showing {{ count }} results per page
+		  {% endblocktrans %}
 	      </span>
 	    </div>
 	    <div class="pagination-right js-pagination-next">
 	      {% if formtable.page.has_next %}
-	      <a href="?page={{ formtable.page.next_page_number }}">&gt;&gt;&gt;</a>
+	      <a href="?page={{ formtable.page.next_page_number }}">{% trans "&gt;&gt;&gt;" %}</a>
```
Comment (dwaynebailey) [inline pootle/templates/includes/formtable.html]: These need to be translated why?

File: pootle/templates/includes/formtable.html [hunk 2]
```diff
@@ -12,24 +12,28 @@
...
 	    </div>
 	    <div class="pagination-center">
 	      <span class="pagination-info js-pagination-page-no">
-		Page {{ formtable.page_no }} of {{ formtable.page.paginator.num_pages }}.
+		  {% blocktrans with page_number=formtable.page_no page_count=formtable.page.paginator.num_pages trimmed %}
+		  Page {{ page_number }} of {{ page_count }}.
+		  {% endblocktrans %}
 	      </span>
 	      <span class="pagination-info js-pagination-items-per-page">
-		{% blocktrans %}Showing{% endblocktrans %} {{ formtable.results_per_page }} {{ formtable.form.results_per_page.label }}
```
Comment (dwaynebailey) [inline pootle/templates/includes/formtable.html]: Ouch @phlax was that you ;)

**Comment by dwaynebailey** [comment]
gtm

File: pootle/templates/includes/formtable.html (hunk 1 above)
Comment (unho) [inline pootle/templates/includes/formtable.html]: I was thinking RTL. Or some language with its quirks regarding how to represent this action. But I will not oppose at all in dropping this. I even put it on a separate commit anticipating this possibility.

**Pull request merged**